### ISSUE TITLE
Missing PropTypes errors

### ISSUE THREAD

**Issue by laksh22** [scorelab/LabelLab#308]
**Describe the bug**
After pulling the latest code, there seem to be a bunch of errors. The initial error was as follows:
```
./src/components/project/label/index.js
  Line 111:17:  Missing "key" prop for element in iterator  react/jsx-key
```

After solving this, the next error was:
```
./src/components/profile/index.js
  Line 141:27:  Missing "key" prop for element in iterator  react/jsx-key
```

After solving the previous error, the PropTypes error is shown:
![eslint](https://user-images.githubusercontent.com/9462834/75603799-e9e42680-5b0c-11ea-96f0-909e5a6aba15.PNG)

Even after adding the proptypes for this component, the same error is shown for more components.

**To Reproduce**
1. Pull the latest code
2. Run LabelLab by running `npm run dev` in `labellab-server`
3. Fix the first error shown by adding `key={index}` to line 112 of `label/index.js`
4. Fix the second error by adding `key={project._id}` to line 142 of `profile/index.js`
5. See the errors

**Expected behavior**
Code should compile without any missing PropTypes errors even if prop types are not present

**Screenshots**
Attached above

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 80

**Additional context**
I think these errors are due to the changes in PR #303 but I'm not sure. I tried to add `"react/prop-types": [0]` to `.eslintrc.js` but it did not solve the issue.

**Comment by laksh22** [comment]
I rolled back my code to `702c84e` and the code seems to work properly. I think this error is occurring due to the react app being ejected.

**Comment by carpecodeum** [comment]
> I rolled back my code to `702c84e` and the code seems to work properly. I think this error is occurring due to the react app being ejected.

@laksh22 react-eject script only ejects the configuration files of the CRA it does not change the code structure as such. What I think is the issue here is that the props-types are required and are not provided. It is a solely prop-type related issue.

**Comment by laksh22** [comment]
@carpecodeum This error shows up when prop-types are not even defined for a component. Additionally, there were errors not related to prop-types which were shown in the beginning, such as the `Missing "key" prop for element in iterator` error which I have mentioned above. These errors did not show up before, even though there has been no change in these components.

**Comment by anjulalk** [comment]
@laksh22 Did you try deleting 'node_modules' and reinstalling? #303 is supposed to fix this issue. There were some dependency updates so clean your cache and do a `npm install`.

**Comment by kushaggarwal** [comment]
Hello ,
I am facing the same error where it describes that **options' is missing in props validation** I have retried after deleting the node modules folder and installing it again but the error still persist 
Can anybody help
Thank You

**Comment by laksh22** [comment]
I'm closing this issue since it has been fixed by #312
